packages/web/src/apollo/index.tsx

```diff
@@ -210,7 +233,35 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
     ...rest,
   })
 
-  return <ApolloProvider client={client}>{children}</ApolloProvider>
+  const extendErrorAndRethrow = (error: any, _errorInfo: React.ErrorInfo) => {
+    error['mostRecentRequest'] = data.mostRecentRequest
+    error['mostRecentResponse'] = data.mostRecentResponse
+    throw error
+  }
```

Comment:
Effectively, if an error is caught via the new [error boundary](https://reactjs.org/docs/error-boundaries.html) below, the most recent res/req info is attached.